php/php.editor/src/org/netbeans/modules/php/editor/model/nodes/MagicMethodDeclarationInfo.java

```diff
@@ -51,10 +52,18 @@
     private String methodName;
     private int offset;
     private int typeOffset;
+    private final boolean isStatic;
 
     MagicMethodDeclarationInfo(PHPDocMethodTag node) {
         super(node);
+        // @method int get(Type $object) message
+        // @method static int staticGet(Type $object) message
         String[] parts = node.getValue().trim().split("\\s+", 3); //NOI18N
+        isStatic = parts.length >= 1 && parts[0].equals("static"); // NOI18N NETBEANS-1861
+        assert isStatic == node.isStatic();
```

Comment:
Consider adding more information to the assert (imagine that it happens - no message will be reported so you know nothing about the failure).